node/src/actor_system_factory.rs

```diff
@@ -378,7 +378,17 @@ pub trait ActorFactory {
     fn make_and_start_configurator(&self, config: &BootstrapperConfig) -> ConfiguratorSubs;
 }
 
-pub struct ActorFactoryReal {}
+pub struct ActorFactoryReal {
+    logger: Logger,
+}
+
+impl ActorFactoryReal {
+    pub fn new() -> Self {
+        Self {
+            logger: Logger::new("ActorFactory"),
```

Comment:
Some of the tests in this file should perform the thorough sequence of actor startups. One moment in that process should belong to the BlockchainBridge creation and startup. Those actors alone are actually all just mocked figures.

However, you added a Logger there which is not meant for an actor, it is meant to log messages before the startups, in the common area. That leads me to the idea that this logging must be also performed during such tests. First I thought that you got yourself into a situation which is so hard for testing that I'd spare you from being bothered about it. Later I realised that it shouldn't be such a big deal, also for the reasons exhibited previously. 

So I'd wish you could identify a proper test in the suit in this file and add in one log assertion that begins like "INFO: ActorFactory: Blockchain service url has been set to " and continues with the actual host. Can you please complete the assertion to the full? 

(Moral: It would expose us to just a marginal risk and damage but theoretically you could have named the Logger somehow silly, or in a misleading way. This test should look after this problematics.)

Reply:
JUST NICE TO HAVE